Gemfile

```diff
@@ -101,6 +101,8 @@ gem 'awesome_nested_fields'
 # A modern, responsive front-end-framework based on Material Design
 gem 'materialize-sass'
 
+gem 'faker'
```

Comment:
please add a short description of what this gem does and preferably a repository link for further documentation

Reply:
Done